### ISSUE TITLE
Fix webgl sample

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/html5rocks#245]
```
Sample is not loading due to an update of arrayBuffer method.
An updated version might be available in Khronos sample list.
```

Original issue reported on code.google.com by `erne...@google.com` on 25 Oct 2010 at 6:40

**Comment by GoogleCodeExporter** [comment]
Original comment by `ericbide...@html5rocks.com` on 7 Nov 2010 at 11:47

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision 0aa04dfeda.
```

Original comment by `ericbide...@html5rocks.com` on 7 Nov 2010 at 11:49
- Changed state: **Fixed**
